pkg/flexvolume/client.go

```diff
@@ -22,43 +39,119 @@ func (c *client) Init() error {
 }
 
 func (c *client) Attach(jsonOptions map[string]string) error {
+	var output []byte
 	_, err := c.apiClient.Attach(
 		context.Background(),
 		&AttachRequest{
 			JsonOptions: jsonOptions,
 		},
 	)
+	if err == nil {
+		a := AttachSuccessOutput{
+			Status: "Success",
+			Device: jsonOptions["volumeID"],
+		}
+		output, _ = json.Marshal(a)
+	} else {
+		a := FailureOutput{
+			Status:  "Failure",
+			Message: err.Error(),
+		}
+		output, _ = json.Marshal(a)
+	}
+
+	os.Stdout.Write(output)
 	return err
 }
 
 func (c *client) Detach(mountDevice string) error {
+	var output []byte
+
 	_, err := c.apiClient.Detach(
 		context.Background(),
 		&DetachRequest{
 			MountDevice: mountDevice,
 		},
 	)
+
+	if err == nil {
+		a := SuccessOutput{
```

Comment:
success output can just be a constant

```go
var (
  successBytes = []byte(`{"Status":"Success"}`)
)
```

func init() {
  successBytes_, err := json.Marshal(
    map[string]
  )
}
```